README.md

```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the 
+[Python APT library page](https://launchpad.net/ubuntu/+source/python-apt) 
+and find which source file you have to use according to your system (e.g. 
+for Impish it's `python-apt_2.2.1.tar.xz`). So the instructions (including 
+virtualenv creation and activation) would be:
```

Comment:
I would leave out `(including virtualenv creation and activation)` as this is obvious when you look at the instructions, and it just makes the text harder to read.

Reply:
Simpler is better.